tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "noImplicitAny": false,
     "strictBindCallApply": false,
     "forceConsistentCasingInFileNames": false,
-    "noFallthroughCasesInSwitch": false
+    "noFallthroughCasesInSwitch": false,
+    "esModuleInterop": true
```

Comment:
please add `strict: true` to the tsconfig. 
It'd actually be best if you can just copy the tsconfig from the backend template and put it here.